thunder/executors/nvfuserex_impl.py

```diff
@@ -2633,5 +2633,4 @@ def embedding(
     return fd.ops.embedding_fwd(*nv_inputs)
 
 
-register_supported(PrimIDs.EMBEDDING, embedding, _embedding_check)
```

Comment:
Re https://github.com/Lightning-AI/lightning-thunder/pull/1689#issuecomment-2613090315
Thank you, Riccardo, the requested change is part of the PR.